compiler/src/iree/compiler/GlobalOptimization/test/expand_vectors.mlir

```diff
@@ -0,0 +1,95 @@
+// RUN: iree-opt --iree-global-opt-expand-vectors --cse --split-input-file %s | FileCheck %s
```

Comment:
I think we don't need `--cse` in the test. The pattern is straight-forward enough, and we don't need to run such cleanups in the tests.